ansys/mapdl/core/launcher.py

```diff
@@ -586,7 +585,7 @@ def save_ansys_path(exe_loc=''):
     with open(CONFIG_FILE, 'w') as f:
         print('Cached ANSYS executable not found')
         try:
```

Comment:
Just remove the entire try except block here.

Reply:
Done